apps/remote_control/lib/lexical/remote_control/code_action/handlers/organize_aliases.ex

```diff
@@ -37,9 +40,32 @@ defmodule Lexical.RemoteControl.CodeAction.Handlers.OrganizeAliases do
   end
 
   defp check_aliases(%Document{}, %Analysis{} = analysis, %Range{} = range) do
-    case Analysis.module_scope(analysis, range) do
-      %Scope{aliases: [_ | _]} -> :ok
-      _ -> :error
+    with %Scope{aliases: [_ | _]} <- Analysis.module_scope(analysis, range),
+         true <-
+           ancestor_is_alias?(analysis, range.start) or
+             token_at_cursor_is_alias?(analysis, range.start) do
+      :ok
+    else
+      _ ->
+        :error
```

Comment:
Perhaps we don't need to check whether the current module scope has aliases, since we're explicitly looking for an alias ancestor or current token?

```suggestion
    if ancestor_is_alias?(analysis, range.start) or token_at_cursor_is_alias?(analysis, range.start) do
      :ok
    else
      :error
    end
```